docs/langref/relay_op.rst

```diff
@@ -196,6 +196,16 @@ This level support backpropagation of broadcast operators. It is temporary.
    tvm.relay.contrib.adaptive_avg_pool2d
 
 
+**Level 11: QNN Dialect Operators**
+
+This level supports quantized operators present in the QNN dialect.
+
+.. autosummary::
+   :nosignatures:
+
+   tvm.relay.qnn.op.requantize
+
+
```

Comment:
I have more nits I'm sorry :) 
Extra newline.

Reply:
I think this one seems fine. I just followed the style in this file. There are 2 new lines between each section.